ShovIP/app/src/main/java/com/example/shovip/DialPadFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.shovip
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.example.shovip.databinding.FragmentDialPadBinding
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DialPadFragment : Fragment() {
+
+    private var _binding: FragmentDialPadBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        _binding = FragmentDialPadBinding.inflate(inflater, container, false)
+        return binding.root
+
```

Comment:
This is just about code style, but I would recommend to avoid unnecessary empty lines.